src/main/java/codeu/model/data/User.java

```diff
@@ -24,6 +24,7 @@
   private final String passwordHash;
   private final Instant creation;
   private String bio;
+  private Boolean admin;
 
```

Comment:
just clarifying here but this is a Boolean object rather than boolean primitive type because you store it as a String in DataStore, so converting back it has to be an object, right?

Reply:
That's correct!